src/components/MoreInformation.js

```diff
@@ -20,6 +21,9 @@ const useStyles = makeStyles((theme) => ({
         display: "block",
         "padding-bottom": theme.spacing(1),
     },
+    formattedExtraData: {
+        color: "#AD003B",
```

Comment:
Can we change this to something from the material UI theme? Maybe theme.palette.error.dark?